BDArmory/Damage/ModuleDrainEC.cs

```diff
@@ -158,14 +166,14 @@ void UpdateEMPLevel()
                 var message = "Rebooting " + vessel.vesselName;
                 if (BDArmorySettings.DEBUG_DAMAGE) Debug.Log("[BDArmory.ModuleDrainEC]: " + message);
                 BDACompetitionMode.Instance.competitionStatus.Add(message);
-                EnableVessel(6);
+                EnableVessel((EMPbuildupTiers)Enum.ToObject(typeof(EMPbuildupTiers), totalEMPTiers));
```

Comment:
You could introduce
```csharp
static readonly EMPbuildupTiers EMPTierMax = Enum.GetValues(typeof(EMPbuildupTiers)).Cast<EMPbuildupTiers>().Max();
```
after the `enum EMPbuildupTiers` definition and then use
```csharp
EnableVessel(EMPTierMax);
```
here instead.